### ISSUE TITLE
Split into few charts

### ISSUE THREAD

**Issue by SkeLLLa** [WeblateOrg/helm#369]
### Describe the problem

Currently there's one chart that has postgres and redis dependencies.
Values for standalone installation with external databased are mixed up and from time to time cause upgrade problems.

### Describe the solution you'd like

I'd suggest splitting the chart into two, like it's done in many apps like prometheus, grafana, etc.

The first chart would be standalone weblate, without any deps and with configuration that will look more or less the same to current with external databases.
The second one could be called `weblate-stack` with dependencies to the first "standalone" chart and database charts dependencies.

In this way it will be much easier to maintain main chart and will allow not to adjust it's config to some third-parties that may change their values structure (e.g. like it was with bitnami charts)

**Comment by nijel** [comment]
This sounds reasonable, but I'm not really an expert here, see https://github.com/WeblateOrg/helm/issues/348

**Comment by SkeLLLa** [comment]
Yeah. I've saw that. Maybe at some point I'll have a time, but I'm actually also not an expert in charts. Or maybe others will see that and will make a contribution.
